src/pages/ProfilePage.js

```diff
@@ -145,7 +142,7 @@ function ProfilePage(props) {
 
     return (
         <ScreenWrapper testID={ProfilePage.displayName}>
-            <FullPageNotFoundView shouldShow={CONST.RESTRICTED_ACCOUNT_IDS.includes(accountID)}>
+            <FullPageNotFoundView shouldShow={_.isEmpty(login) || shouldShowBlockingView || CONST.RESTRICTED_ACCOUNT_IDS.includes(accountID)}>
```

Comment:
Why did you add `_.isEmpty(login)`? It causes regression

Before:
<img width="1183" alt="Screenshot 2024-02-21 at 7 09 24 PM" src="https://github.com/Expensify/App/assets/108292595/2f5a5a64-f1c0-4f8b-9ba4-c0db502921f4">

After:

<img width="1183" alt="Screenshot 2024-02-21 at 7 09 45 PM" src="https://github.com/Expensify/App/assets/108292595/e040f072-8b58-42c0-8afd-2b47c08b2889">